src/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300px;
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
+  outline: none;
+  box-shadow: 0 1px 8px white;
+  background: transparent;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.label {
+  display: block;
+  margin: 20px 8px 0;
+}
+
+.search__big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+}
+
+.search__small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px 11px;
+  background-position: 12px 12px;
+}
+
+.search:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
```suggestion
  font-weight: bold;
```

Reply:
I don't see that you've committed your changes, probably – you've forgotten to run `git commit` and `git push` commands